content/ecs_advance_schedule/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Advanced Scheduling techniques on ECS
+chapter: false
+weight: 8
```

Comment:
Can you change the weight here to be right after the capacity providers chapter? Right now it's up too high in the list of chapters.